webui/react/src/pages/Cluster/ClusterOverallBar.tsx

```diff
@@ -2,14 +2,17 @@ import React, { useMemo } from 'react';
 
 import Section from 'components/Section';
 import SlotAllocationBar from 'components/SlotAllocationBar';
-import { useStore } from 'contexts/Store';
 import Message, { MessageType } from 'shared/components/Message';
+import { initClusterOverview, useAgents, useClusterOverview } from 'stores/agents';
 import { ShirtSize } from 'themes';
 import { ResourceType } from 'types';
 import { getSlotContainerStates } from 'utils/cluster';
+import { Loadable } from 'utils/loadable';
 
 export const ClusterOverallBar: React.FC = () => {
-  const { agents, cluster: overview } = useStore();
+  const overview = Loadable.getOrElse(initClusterOverview, useClusterOverview());
+  // TODO: handle loading state
```

Comment:
I honestly just couldn't figure out the correct loading state for this one (without writing a bunch of new CSS). Thought about having the whole page load but the loading state is only in one tab and I didn't see a way to have just the one tab loading (again, without writing a bunch of new code).

Reply:
I guess we could make `overview` a required prop and conditionally render this?